dashboard/db/migrate/20250207175343_add_foreign_key_from_games_to_backpacks_and_backfill.rb

```diff
@@ -0,0 +1,20 @@
+class AddForeignKeyFromGamesToBackpacksAndBackfill < ActiveRecord::Migration[6.1]
+  def change
+    add_column :backpacks, :game_id, :integer, index: true
+    add_foreign_key :backpacks, :games, column: :game_id
+
+    reversible do |dir|
+      dir.up do
+        game_id = Game.find_by(name: 'Javalab')&.id
+        if game_id
+          Backpack.where(game_id: nil).update_all(game_id: game_id)
```

Comment:
I'm generally wary of the performance impacts of backfills being executed within schema migrations, but this one seems low risk, particularly given the small size of the table.